code/modules/mob/living/carbon/human/human_attackhand.dm

```diff
@@ -179,9 +179,9 @@
 			if (H.stats.getStat(STAT_ROB) >= 80)
 				var softcap = H.stats.getStat(STAT_ROB) / 10
 				var newrob = (72 + softcap) / 10
-				stat_damage = 3 + max(0, newrob)
+				stat_damage = 3 + max(0, newrob) + H.punch_damage_increase // This makes knuckledusters actually work since NOTHING called this var, ever
```

Comment:
```suggestion
				stat_damage = 3 + max(0, newrob)
```
Dont doble punch increases

Reply:
I've tested this live, in many situations, with both dummies and roaches, this doesn't double punching damage, it caused the exact damage a normal punch at 40 ROB with knuckledusters SHOULD give
This var is called nowhere at all unless you put on the gloves that affect said var